akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Routers.scala

```diff
@@ -181,4 +181,10 @@ abstract class PoolRouter[T] extends DeferredBehavior[T] {
    * Set the props used to spawn the pool's routees
    */
   def withRouteeProps(routeeProps: Props): PoolRouter[T]
+
+  /**
+   * install a predicate that identifies messages intended to be broadcasted to all routees.
+   * @param pred paredicate used to determine if a message is to be broadcasted or not.
```

Comment:
Maybe skip the two lines parts and go for one single with what it does "Any message that the predicate returns true for will be broadcast to all routees"